pallets/subtensor/src/lib.rs

```diff
@@ -262,6 +266,11 @@ pub mod pallet {
     pub fn TotalSupply<T: Config>() -> u64 {
         21_000_000_000_000_000 // Rao => 21_000_000 Tao
     }
+    /// Hotkey swap cost.
+    #[pallet::type_value]
+    pub fn HotkeySwapCost<T: Config>() -> u64 {
```

Comment:
Not sure this is actually being used? Seems like it is using `InitialHotkeySwapCost` as the default instead.

Reply:
good catch! will fix

Reply:
done